frontend/subscriptions.js

```diff
@@ -67,9 +69,15 @@ export default (subscribe) => {
       top: 'var(--safe-area-inset-top) !important',
       paddingTop: 'var(--safe-area-inset-top) !important',
       paddingBottom: 'calc(var(--safe-area-inset-bottom) * 2) !important',
-      background: 'rgb(34, 77, 143)',
     });
 
+    if (!isTablet) {
+      css.global(`iframe[title="${iframeChatTitle || ''}"]`, {
```

Comment:
as discussed, i would also add a background color to the iframe. depending on the shop setup, elements from the page behind the overlay can shine through.

in my case i have a black navigation bar configured on start page, and a floating iOS theme TabBar.

![Bildschirmfoto 2025-05-26 um 11 27 06](https://github.com/user-attachments/assets/e6c70b22-e32d-4c47-b7a3-6c4efc1fdf71)